src/acachecontrol/cache.py

```diff
@@ -5,20 +5,39 @@
 import asyncio
 import hashlib
 import json
+import logging
 import time
 from typing import Any, Dict, Tuple
 
 from .exceptions import CacheException, TimeoutException
 
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_MAX_AGE = 120  # seconds
+
 
 class AsyncCache:
-    def __init__(self):
+    def __init__(self, config={}):
         self.cache = {}  # type: Dict
         self._wait_until_completed = set()
+        self.default_max_age = config.get("max-age", DEFAULT_MAX_AGE)
 
     def __contains__(self, key):
         return self._make_key_hashable(key) in self.cache
```

Comment:
```suggestion
```
It looks redundant since `some_key in cache` statement has no value now. The only method that is required is `has_valid_entry()`.

Reply:
Probably yes, and of course it was logically to test key against cache with "in" operator, that's why everything was inside __contains__. WDYT?